index.html

```diff
@@ -39,28 +41,41 @@ <h1 class="sr-only">Bikes for Refugees Scotland - Providing donated bikes and ac
 							<a class="nav-link" href="#">Contact Us</a>
 						</li>
 					</ul>
-					<a href="#" class="btn btn-primary ml-auto">
-						Donate Now
-					</a>
 				</div>
 			</div>
 		</div>
+              </div>
+              <div class="col-md-2 my-col">
+                  <a href="#" class="btn btn-primary ml-auto">
+                      Donate Now
+                  </a>
+              </div>
+              </div>
+              <div class="row">
+                <div class="col">
+                    <div class="alert alert-info" role="alert">
+                        You've helped us give <span id="donation-count-alert">72</span> bikes to refugees and asylum seekers. <strong>Thank you!</strong>
+                    </div>
+                </div>
+		<a href="#content" class="sr-only sr-only-focusable">Skip to main content</a>
 
 		<div id="content" class="container site-content">
-			<div class="alert alert-info" role="alert">
-				You've helped us give <span id="donation-count-alert">72</span> bikes to refugees and asylum seekers. <strong>Thank you!</strong>
-			</div>
 			<div class="row">
 				<div class="primary-content col-lg-8">
 					<div class="jumbotron">
 						<h1 class="display-3">Bikes for Refugees</h1>
-						<p class="lead">Providing donated bikes and accessories to refugees and asylum seekers in Scotland.</p>
+						<p class="lead">Providing donated bikes and accessories to refugees and asylum seekers in london.</p>
 						<hr class="my-4">
 						<div class="buttons">
-							<a class="btn btn-primary btn-lrg" href="#">Donate a bike today</a>
-							<a class="btn btn-secondary btn-lrg" href="#">Volunteer</a>
+							<a class="btn btn-lrg blue-button" href="#">Donate a bike today</a>
+							<a class="btn volunteer btn-lrg" href="#">Volunteer</a>
 						</div>
-					</div>
+          </div>
+          <div class="parentBox">
+            <div class="box"> check availability </div>
```

Comment:
These look good. Just be careful with your choice of names. Be more descriptive than `box`, for example `feature` or `highlight` or `action-box`. And for parent elements, if they have no other use than wrapping around, you can call them `...-wrapper`, for example `feature-wrapper`